### PR TITLE
[Fix] French starts using plural at 2

### PR DESCRIPTION
**Pull request by FredericEspiau**
In French, plural mark starts from 2

Source: https://leconjugueur.lefigaro.fr/uk_pluriel_debut.php

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ var defaultPluralRules = {
     bosnian_serbian: russianPluralGroups,
     chinese: function () { return 0; },
     croatian: russianPluralGroups,
-    french: function (n) { return n > 1 ? 1 : 0; },
+    french: function (n) { return n >= 2 ? 1 : 0; },
```
Comment (ljharb) [inline index.js:61]: `> 1` and `>= 2` are identical for integers. is there a need to change this?

**Review by ljharb — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (piitaya) [reply index.js:61]: 1.99 is singular in french. 2 is plural. ☺️

Comment (FredericEspiau) [reply index.js:61]: Yup, this is to handle non-integers between 1 and 2 @ljharb

Comment (ljharb) [reply index.js:61]: This api only expects integers.

Comment (FredericEspiau) [reply index.js:61]: Well this is unexpected, should I have a go at it or this is unwanted ?

Comment (ljharb) [reply index.js:61]: No, this seems a fine fix, even if it’s not needed for the integer case. I’ll review it tomorrow.

Comment (FredericEspiau) [reply index.js:61]: Alright, thanks !

**Review by ljharb — APPROVED** [review]

**Pull request merged**